assets/css/less/admin-notice-review.less

```diff
@@ -0,0 +1,112 @@
+.wpbdp-review-notice {
+	&.notice {
+		border: 1px solid #ddd;
+		text-align: center;
+		position: fixed !important;
+		z-index: 999;
+		bottom: 0;
+		right: 0;
+		width: 400px;
+		margin-bottom: 0;
+		padding-bottom: 15px;
+
+		form, p {
+			font-size: 14px;
+			max-width: 600px;
+			margin-left: auto;
+			margin-right: auto;
+		}
+	}
+	a {
+		margin: 5px;
+	}
+	.wpbdp-button {
+		&-primary, &-secondary{
+			text-shadow: none;
+			box-shadow: none;
+			border-radius: 30px;
+			border:1px solid @wpbdp-primary-button-color;
+			font-size: 1em;
+			transition: all .2s ease;
+			height: 28px;
+			min-height: 28px;
+			outline: none;
+			line-height: 26px;
+			min-width: 100px;
+			display: inline-block;
+			text-align: center;
+			padding: 2px 10px;
+			text-decoration: none;
+		}
+		&-primary{
+			background-color: @wpbdp-primary-button-color !important;
+			color: #fff !important;
+			&:hover, &:focus{
+				background: @wpbdp-primary-button-hover !important;
+				border-color: @wpbdp-primary-button-hover !important;
+			}
+		}
+		&-secondary{
+			color: @wpbdp-primary-button-color;
+			background-color: transparent;
+			&:hover, &:focus{
+				border-color: @wpbdp-primary-button-hover !important;
+				background: @wpbdp-primary-button-hover !important;
+				color: #fff;
+				box-shadow: none;
+			}
+		}
+	}
+	p > span {
+		font-size: 13px;
+		opacity: 0.75;
+	}
+	.wpbdp_error, label {
+		color: #444;
+		text-align: left;
+	}
+	label {
+		font-size: 14px;
+	}
+	.wpbdp-wait{
+		margin: 20px;
+		width: 20px;
+		height: 20px;
+		position: relative;
+		display: inline-block;
+		&:before{
+			content: '';
+			display: inline-block;
+
+			position: absolute;
+			background: transparent;
+			border: 1px solid #fff;
+			border-top-color: transparent;
+			border-left-color: transparent;
+			border-bottom-color: @wpbdp-primary-button-hover;
+			border-right-color: @wpbdp-primary-button-hover;
+			border-radius: 50%;
```

Comment:
Some lines in this block look like duplicated with https://github.com/Strategy11/business-directory-plugin/blob/master/assets/css/less/admin.less#L466

I think there would be a way to reuse them. I am not quite familiar with LESS.

Please add space before `{` as well.

Reply:
The less will be cleaned later. There are too many variables that can be reused and the layout does not really follow the less rules. For example, I think global common variables should all be in one file

Reply:
@theRealRizeo It's ok.